LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 Avishay Bar
+Copyright (c) 2020 Aristide Bouix, based on the work of Avishay Bar
```

Comment:
Why are you copywriting my work? :\

I can mention you on the credits section if you like, but the copyright should stick.

Reply:
Indeed, this is only for my local fork as I've made a few modifications. No problem with rejecting this change on the master.